C/tests/ParserTest_0000_0999.cpp

```diff
@@ -2015,102 +2015,464 @@ void ParserTest::case0279()
 
 void ParserTest::case0280()
 {
-
+    parse("int x ( y ) { return y ; }",
```

Comment:
This test should actually be in `case0200` and `case0279` as this isn't a K&R function.

Also, as the tests here shouldn't contain inside the body of the functions (to avoid confusion/clutter for things that aren't being tested).